scripts/make_tau_sse_2019_index.py

```diff
@@ -6,7 +6,7 @@
 import csv
 from pathlib import Path
 
-DATASET_INDEX_PATH = "../soundata/datasets/indexes/tau_sse_2019_index.json"
+DATASET_INDEX_PATH = "../soundata/datasets/indexes/tau2021sse_nigens_index.json"
```

Comment:
2019 vs 2021?

Reply:
good catch, it got messed by the find and replace and this does not affect the tests